### ISSUE TITLE
rospilot is failing to build on kinetic

### ISSUE THREAD

**Issue by tfoote** [rospilot/rospilot#72]
http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__rospilot__ubuntu_xenial_amd64__binary/53/console

This appears to be an issue on Ubuntu Xenial but not Debian Jessie

It looks to be an npm installation/dependency issue:

```
23:23:02 cd /tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /tmp/binarydeb/ros-kinetic-rospilot-1.3.7 /tmp/binarydeb/ros-kinetic-rospilot-1.3.7 /tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu /tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu /tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu/CMakeFiles/javascript_assets.dir/DependInfo.cmake --color=
23:23:02 Scanning dependencies of target javascript_assets
23:23:02 make[4]: Leaving directory '/tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu'
23:23:02 make -f CMakeFiles/javascript_assets.dir/build.make CMakeFiles/javascript_assets.dir/build
23:23:02 make[4]: Entering directory '/tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu'
23:23:02 [  7%] Updating npm
23:23:02 npm install npm 2> /dev/null
23:23:33 /tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu
23:23:33 ├── ansi-regex@3.0.0 
23:23:33 ├── aproba@1.1.2 
23:23:33 ├─┬ cacache@9.2.9 
23:23:33 │ ├─┬ lru-cache@4.1.1 
23:23:33 │ │ ├── pseudomap@1.0.2 
23:23:33 │ │ └── yallist@2.1.2 
23:23:33 │ └── y18n@3.2.1 
23:23:33 ├── detect-indent@5.0.0 
23:23:33 ├─┬ glob@7.1.2 
23:23:33 │ ├── fs.realpath@1.0.0 
23:23:33 │ ├─┬ minimatch@3.0.4 
23:23:33 │ │ └─┬ brace-expansion@1.1.8 
23:23:33 │ │   ├── balanced-match@1.0.0 
23:23:33 │ │   └── concat-map@0.0.1 
23:23:33 │ └── path-is-absolute@1.0.1 
23:23:33 ├── hosted-git-info@2.5.0 
23:23:33 ├─┬ JSONStream@1.3.1 
23:23:33 │ ├── jsonparse@1.3.1 
23:23:33 │ └── through@2.3.8 
23:23:33 ├─┬ libnpx@9.6.0 
23:23:33 │ ├── dotenv@4.0.0 
23:23:33 │ ├── y18n@3.2.1 
23:23:33 │ └─┬ yargs@8.0.2 
23:23:33 │   ├── camelcase@4.1.0 
23:23:33 │   ├─┬ cliui@3.2.0 
23:23:33 │   │ ├─┬ string-width@1.0.2 
23:23:33 │   │ │ ├── code-point-at@1.1.0 
23:23:33 │   │ │ └─┬ is-fullwidth-code-point@1.0.0 
23:23:33 │   │ │   └── number-is-nan@1.0.1 
23:23:33 │   │ ├─┬ strip-ansi@3.0.1 
23:23:33 │   │ │ └── ansi-regex@2.1.1 
23:23:33 │   │ └── wrap-ansi@2.1.0 
23:23:33 │   ├── decamelize@1.2.0 
23:23:33 │   ├── get-caller-file@1.0.2 
23:23:33 │   ├─┬ os-locale@2.1.0 
23:23:33 │   │ ├─┬ execa@0.7.0 
23:23:33 │   │ │ ├─┬ cross-spawn@5.1.0 
23:23:33 │   │ │ │ └─┬ shebang-command@1.2.0 
23:23:33 │   │ │ │   └── shebang-regex@1.0.0 
23:23:33 │   │ │ ├── get-stream@3.0.0 
23:23:33 │   │ │ ├── is-stream@1.1.0 
23:23:33 │   │ │ ├─┬ npm-run-path@2.0.2 
23:23:33 │   │ │ │ └── path-key@2.0.1 
23:23:33 │   │ │ ├── p-finally@1.0.0 
23:23:33 │   │ │ ├── signal-exit@3.0.2 
23:23:33 │   │ │ └── strip-eof@1.0.0 
23:23:33 │   │ ├─┬ lcid@1.0.0 
23:23:33 │   │ │ └── invert-kv@1.0.0 
23:23:33 │   │ └─┬ mem@1.1.0 
23:23:33 │   │   └── mimic-fn@1.1.0 
23:23:33 │   ├─┬ read-pkg-up@2.0.0 
23:23:33 │   │ ├─┬ find-up@2.1.0 
23:23:33 │   │ │ └─┬ locate-path@2.0.0 
23:23:33 │   │ │   ├─┬ p-locate@2.0.0 
23:23:33 │   │ │   │ └── p-limit@1.1.0 
23:23:33 │   │ │   └── path-exists@3.0.0 
23:23:33 │   │ └─┬ read-pkg@2.0.0 
23:23:33 │   │   ├─┬ load-json-file@2.0.0 
23:23:33 │   │   │ ├─┬ parse-json@2.2.0 
23:23:33 │   │   │ │ └─┬ error-ex@1.3.1 
23:23:33 │   │   │ │   └── is-arrayish@0.2.1 
23:23:33 │   │   │ ├── pify@2.3.0 
23:23:33 │   │   │ └── strip-bom@3.0.0 
23:23:33 │   │   └─┬ path-type@2.0.0 
23:23:33 │   │     └── pify@2.3.0 
23:23:33 │   ├── require-directory@2.1.1 
23:23:33 │   ├── require-main-filename@1.0.1 
23:23:33 │   ├── set-blocking@2.0.0 
23:23:33 │   ├─┬ string-width@2.1.1 
23:23:33 │   │ └── is-fullwidth-code-point@2.0.0 
23:23:33 │   ├── which-module@2.0.0 
23:23:33 │   └── yargs-parser@7.0.0 
23:23:33 ├─┬ lru-cache@4.1.1 
23:23:33 │ ├── pseudomap@1.0.2 
23:23:33 │ └── yallist@2.1.2 
23:23:33 ├── meant@1.0.0 
23:23:33 ├─┬ node-gyp@3.6.2 
23:23:33 │ ├── fstream@1.0.11 
23:23:33 │ ├─┬ minimatch@3.0.4 
23:23:33 │ │ └─┬ brace-expansion@1.1.8 
23:23:33 │ │   ├── balanced-match@1.0.0 
23:23:33 │ │   └── concat-map@0.0.1 
23:23:33 │ ├── nopt@3.0.6 
23:23:33 │ ├── semver@5.3.0 
23:23:33 │ └─┬ tar@2.2.1 
23:23:33 │   └── block-stream@0.0.9 
23:23:33 ├─┬ normalize-package-data@2.4.0 
23:23:33 │ └─┬ is-builtin-module@1.0.0 
23:23:33 │   └── builtin-modules@1.1.1 
23:23:33 ├─┬ npm@5.4.1 
23:23:33 │ ├── abbrev@1.1.0 
23:23:33 │ ├── ansicolors@0.3.2 
23:23:33 │ ├── ansistyles@0.1.3 
23:23:33 │ ├── archy@1.0.0 
23:23:33 │ ├── bluebird@3.5.0 
23:23:33 │ ├── call-limit@1.1.0 
23:23:33 │ ├── chownr@1.0.1 
23:23:33 │ ├── cmd-shim@2.0.2 
23:23:33 │ ├─┬ columnify@1.5.4 
23:23:33 │ │ ├─┬ strip-ansi@3.0.1 
23:23:33 │ │ │ └── ansi-regex@2.1.1 
23:23:33 │ │ └─┬ wcwidth@1.0.1 
23:23:33 │ │   └─┬ defaults@1.0.3 
23:23:33 │ │     └── clone@1.0.2 
23:23:33 │ ├─┬ config-chain@1.1.11 
23:23:33 │ │ └── proto-list@1.2.4 
23:23:33 │ ├── debuglog@1.0.1 
23:23:33 │ ├─┬ dezalgo@1.0.3 
23:23:33 │ │ └── asap@2.0.5 
23:23:33 │ ├── editor@1.0.0 
23:23:33 │ ├── fs-vacuum@1.2.10 
23:23:33 │ ├── fs-write-stream-atomic@1.0.10 
23:23:33 │ ├── graceful-fs@4.1.11 
23:23:33 │ ├── has-unicode@2.0.1 
23:23:33 │ ├── iferr@0.1.5 
23:23:33 │ ├── imurmurhash@0.1.4 
23:23:33 │ ├── inflight@1.0.6 
23:23:33 │ ├── inherits@2.0.3 
23:23:33 │ ├── ini@1.3.4 
23:23:33 │ ├─┬ init-package-json@1.10.1 
23:23:33 │ │ └── promzard@0.3.0 
23:23:33 │ ├── lazy-property@1.0.0 
23:23:33 │ ├── lockfile@1.0.3 
23:23:33 │ ├── lodash._baseindexof@3.1.0 
23:23:33 │ ├─┬ lodash._baseuniq@4.6.0 
23:23:33 │ │ ├── lodash._createset@4.0.3 
23:23:33 │ │ └── lodash._root@3.0.1 
23:23:33 │ ├── lodash._bindcallback@3.0.1 
23:23:33 │ ├── lodash._cacheindexof@3.0.2 
23:23:33 │ ├── lodash._createcache@3.1.2 
23:23:33 │ ├── lodash._getnative@3.9.1 
23:23:33 │ ├── lodash.clonedeep@4.5.0 
23:23:33 │ ├── lodash.restparam@3.6.1 
23:23:33 │ ├── lodash.union@4.6.0 
23:23:33 │ ├── lodash.uniq@4.5.0 
23:23:33 │ ├── lodash.without@4.4.0 
23:23:33 │ ├─┬ mississippi@1.3.0 
23:23:33 │ │ ├─┬ concat-stream@1.6.0 
23:23:33 │ │ │ └── typedarray@0.0.6 
23:23:33 │ │ ├─┬ duplexify@3.5.0 
23:23:33 │ │ │ ├─┬ end-of-stream@1.0.0 
23:23:33 │ │ │ │ └── once@1.3.3 
23:23:33 │ │ │ └── stream-shift@1.0.0 
23:23:33 │ │ ├── end-of-stream@1.4.0 
23:23:33 │ │ ├── flush-write-stream@1.0.2 
23:23:33 │ │ ├── from2@2.3.0 
23:23:33 │ │ ├─┬ parallel-transform@1.1.0 
23:23:33 │ │ │ └── cyclist@0.2.2 
23:23:33 │ │ ├── pump@1.0.2 
23:23:33 │ │ ├── pumpify@1.3.5 
23:23:33 │ │ ├─┬ stream-each@1.2.0 
23:23:33 │ │ │ └── stream-shift@1.0.0 
23:23:33 │ │ └─┬ through2@2.0.3 
23:23:33 │ │   └── xtend@4.0.1 
23:23:33 │ ├─┬ mkdirp@0.5.1 
23:23:33 │ │ └── minimist@0.0.8 
23:23:33 │ ├─┬ move-concurrently@1.0.1 
23:23:33 │ │ ├── copy-concurrently@1.0.3 
23:23:33 │ │ └── run-queue@1.0.3 
23:23:33 │ ├── nopt@4.0.1 
23:23:33 │ ├── npm-cache-filename@1.0.2 
23:23:33 │ ├── npm-install-checks@3.0.0 
23:23:33 │ ├── once@1.4.0 
23:23:33 │ ├── opener@1.4.3 
23:23:33 │ ├─┬ osenv@0.1.4 
23:23:33 │ │ ├── os-homedir@1.0.2 
23:23:33 │ │ └── os-tmpdir@1.0.2 
23:23:33 │ ├── path-is-inside@1.0.2 
23:23:33 │ ├── promise-inflight@1.0.1 
23:23:33 │ ├─┬ read@1.0.7 
23:23:33 │ │ └── mute-stream@0.0.7 
23:23:33 │ ├── read-cmd-shim@1.0.1 
23:23:33 │ ├─┬ read-installed@4.0.3 
23:23:33 │ │ └── util-extend@1.0.3 
23:23:33 │ ├── readdir-scoped-modules@1.0.2 
23:23:33 │ ├── retry@0.10.1 
23:23:33 │ ├── rimraf@2.6.1 
23:23:33 │ ├── sha@2.0.1 
23:23:33 │ ├── slide@1.1.6 
23:23:33 │ ├── sorted-object@2.0.1 
23:23:33 │ ├─┬ sorted-union-stream@2.1.3 
23:23:33 │ │ ├─┬ from2@1.3.0 
23:23:33 │ │ │ └─┬ readable-stream@1.1.14 
23:23:33 │ │ │   ├── core-util-is@1.0.2 
23:23:33 │ │ │   ├── isarray@0.0.1 
23:23:33 │ │ │   └── string_decoder@0.10.31 
23:23:33 │ │ └─┬ stream-iterate@1.2.0 
23:23:33 │ │   └── stream-shift@1.0.0 
23:23:33 │ ├── text-table@0.2.0 
23:23:33 │ ├── uid-number@0.0.6 
23:23:33 │ ├── umask@1.1.0 
23:23:33 │ ├─┬ unique-filename@1.1.0 
23:23:33 │ │ └── unique-slug@2.0.0 
23:23:33 │ ├── unpipe@1.0.0 
23:23:33 │ ├─┬ validate-npm-package-license@3.0.1 
23:23:33 │ │ ├─┬ spdx-correct@1.0.2 
23:23:33 │ │ │ └── spdx-license-ids@1.2.2 
23:23:33 │ │ └── spdx-expression-parse@1.0.4 
23:23:33 │ ├─┬ validate-npm-package-name@3.0.0 
23:23:33 │ │ └── builtins@1.0.3 
23:23:33 │ └── wrappy@1.0.2 
23:23:33 ├── npm-lifecycle@1.0.2 
23:23:33 ├── npm-package-arg@5.1.2 
23:23:33 ├─┬ npm-packlist@1.1.8 
23:23:33 │ ├─┬ ignore-walk@3.0.0 
23:23:33 │ │ └─┬ minimatch@3.0.4 
23:23:33 │ │   └─┬ brace-expansion@1.1.8 
23:23:33 │ │     ├── balanced-match@1.0.0 
23:23:33 │ │     └── concat-map@0.0.1 
23:23:33 │ └── npm-bundled@1.0.3 
23:23:33 ├─┬ npm-registry-client@8.4.0 
23:23:33 │ └─┬ concat-stream@1.6.0 
23:23:33 │   └── typedarray@0.0.6 
23:23:33 ├── npm-user-validate@1.0.0 
23:23:33 ├─┬ npmlog@4.1.2 
23:23:33 │ ├─┬ are-we-there-yet@1.1.4 
23:23:33 │ │ └── delegates@1.0.0 
23:23:33 │ ├── console-control-strings@1.1.0 
23:23:33 │ ├─┬ gauge@2.7.4 
23:23:33 │ │ ├── object-assign@4.1.1 
23:23:33 │ │ ├── signal-exit@3.0.2 
23:23:33 │ │ ├─┬ string-width@1.0.2 
23:23:33 │ │ │ ├── code-point-at@1.1.0 
23:23:33 │ │ │ └─┬ is-fullwidth-code-point@1.0.0 
23:23:33 │ │ │   └── number-is-nan@1.0.1 
23:23:33 │ │ ├─┬ strip-ansi@3.0.1 
23:23:33 │ │ │ └── ansi-regex@2.1.1 
23:23:33 │ │ └── wide-align@1.1.2 
23:23:33 │ └── set-blocking@2.0.0 
23:23:33 ├─┬ pacote@6.0.2 
23:23:33 │ ├─┬ make-fetch-happen@2.5.0 
23:23:33 │ │ ├─┬ agentkeepalive@3.3.0 
23:23:33 │ │ │ └─┬ humanize-ms@1.2.1 
23:23:33 │ │ │   └── ms@2.0.0 
23:23:33 │ │ ├── http-cache-semantics@3.7.3 
23:23:33 │ │ ├─┬ http-proxy-agent@2.0.0 
23:23:33 │ │ │ ├─┬ agent-base@4.1.1 
23:23:33 │ │ │ │ └─┬ es6-promisify@5.0.0 
23:23:33 │ │ │ │   └── es6-promise@4.1.1 
23:23:33 │ │ │ └─┬ debug@2.6.8 
23:23:33 │ │ │   └── ms@2.0.0 
23:23:33 │ │ ├─┬ https-proxy-agent@2.1.0 
23:23:33 │ │ │ ├─┬ agent-base@4.1.1 
23:23:33 │ │ │ │ └─┬ es6-promisify@5.0.0 
23:23:33 │ │ │ │   └── es6-promise@4.1.1 
23:23:33 │ │ │ └─┬ debug@2.6.8 
23:23:33 │ │ │   └── ms@2.0.0 
23:23:33 │ │ ├─┬ node-fetch-npm@2.0.2 
23:23:33 │ │ │ ├─┬ encoding@0.1.12 
23:23:33 │ │ │ │ └── iconv-lite@0.4.18 
23:23:33 │ │ │ └── json-parse-better-errors@1.0.1 
23:23:33 │ │ └─┬ socks-proxy-agent@3.0.0 
23:23:33 │ │   ├─┬ agent-base@4.1.1 
23:23:33 │ │   │ └─┬ es6-promisify@5.0.0 
23:23:33 │ │   │   └── es6-promise@4.1.1 
23:23:33 │ │   └─┬ socks@1.1.10 
23:23:33 │ │     ├── ip@1.1.5 
23:23:33 │ │     └── smart-buffer@1.1.15 
23:23:33 │ ├─┬ minimatch@3.0.4 
23:23:33 │ │ └─┬ brace-expansion@1.1.8 
23:23:33 │ │   ├── balanced-match@1.0.0 
23:23:33 │ │   └── concat-map@0.0.1 
23:23:33 │ ├── npm-pick-manifest@1.0.4 
23:23:33 │ ├─┬ promise-retry@1.1.1 
23:23:33 │ │ └── err-code@1.1.2 
23:23:33 │ └─┬ protoduck@4.0.0 
23:23:33 │   └── genfun@4.0.1 
23:23:33 ├─┬ read-package-json@2.0.12 
23:23:33 │ ├── json-parse-better-errors@1.0.1 
23:23:33 │ └── slash@1.0.0 
23:23:33 ├── read-package-tree@5.1.6 
23:23:33 ├─┬ readable-stream@2.3.3 
23:23:33 │ ├── core-util-is@1.0.2 
23:23:33 │ ├── isarray@1.0.0 
23:23:33 │ ├── process-nextick-args@1.0.7 
23:23:33 │ ├── string_decoder@1.0.3 
23:23:33 │ └── util-deprecate@1.0.2 
23:23:33 ├─┬ request@2.81.0 
23:23:33 │ ├── aws-sign2@0.6.0 
23:23:33 │ ├── aws4@1.6.0 
23:23:33 │ ├── caseless@0.12.0 
23:23:33 │ ├─┬ combined-stream@1.0.5 
23:23:33 │ │ └── delayed-stream@1.0.0 
23:23:33 │ ├── extend@3.0.1 
23:23:33 │ ├── forever-agent@0.6.1 
23:23:33 │ ├─┬ form-data@2.1.4 
23:23:33 │ │ └── asynckit@0.4.0 
23:23:33 │ ├─┬ har-validator@4.2.1 
23:23:33 │ │ ├─┬ ajv@4.11.8 
23:23:33 │ │ │ ├── co@4.6.0 
23:23:33 │ │ │ └─┬ json-stable-stringify@1.0.1 
23:23:33 │ │ │   └── jsonify@0.0.0 
23:23:33 │ │ └── har-schema@1.0.5 
23:23:33 │ ├─┬ hawk@3.1.3 
23:23:33 │ │ ├── boom@2.10.1 
23:23:33 │ │ ├── cryptiles@2.0.5 
23:23:33 │ │ ├── hoek@2.16.3 
23:23:33 │ │ └── sntp@1.0.9 
23:23:33 │ ├─┬ http-signature@1.1.1 
23:23:33 │ │ ├── assert-plus@0.2.0 
23:23:33 │ │ ├─┬ jsprim@1.4.0 
23:23:33 │ │ │ ├── assert-plus@1.0.0 
23:23:33 │ │ │ ├── extsprintf@1.0.2 
23:23:33 │ │ │ ├── json-schema@0.2.3 
23:23:33 │ │ │ └── verror@1.3.6 
23:23:33 │ │ └─┬ sshpk@1.13.1 
23:23:33 │ │   ├── asn1@0.2.3 
23:23:33 │ │   ├── assert-plus@1.0.0 
23:23:33 │ │   ├── bcrypt-pbkdf@1.0.1 
23:23:33 │ │   ├── dashdash@1.14.1 
23:23:33 │ │   ├── ecc-jsbn@0.1.1 
23:23:33 │ │   ├── getpass@0.1.7 
23:23:33 │ │   ├── jsbn@0.1.1 
23:23:33 │ │   └── tweetnacl@0.14.5 
23:23:33 │ ├── is-typedarray@1.0.0 
23:23:33 │ ├── isstream@0.1.2 
23:23:33 │ ├── json-stringify-safe@5.0.1 
23:23:33 │ ├─┬ mime-types@2.1.15 
23:23:33 │ │ └── mime-db@1.27.0 
23:23:33 │ ├── oauth-sign@0.8.2 
23:23:33 │ ├── performance-now@0.2.0 
23:23:33 │ ├── qs@6.4.0 
23:23:33 │ ├── stringstream@0.0.5 
23:23:33 │ ├─┬ tough-cookie@2.3.2 
23:23:33 │ │ └── punycode@1.4.1 
23:23:33 │ └── tunnel-agent@0.6.0 
23:23:33 ├── safe-buffer@5.1.1 
23:23:33 ├── semver@5.4.1 
23:23:33 ├── ssri@4.1.6 
23:23:33 ├─┬ strip-ansi@4.0.0 
23:23:33 │ └── ansi-regex@3.0.0 
23:23:33 ├─┬ tar@4.0.1 
23:23:33 │ ├── minipass@2.2.1 
23:23:33 │ ├── minizlib@1.0.3 
23:23:33 │ └── yallist@3.0.2 
23:23:33 ├─┬ update-notifier@2.2.0 
23:23:33 │ ├─┬ boxen@1.1.0 
23:23:33 │ │ ├── ansi-align@2.0.0 
23:23:33 │ │ ├── camelcase@4.1.0 
23:23:33 │ │ ├── cli-boxes@1.0.0 
23:23:33 │ │ ├─┬ string-width@2.1.0 
23:23:33 │ │ │ ├── is-fullwidth-code-point@2.0.0 
23:23:33 │ │ │ └── strip-ansi@4.0.0 
23:23:33 │ │ ├─┬ term-size@0.1.1 
23:23:33 │ │ │ └─┬ execa@0.4.0 
23:23:33 │ │ │   ├── cross-spawn-async@2.2.5 
23:23:33 │ │ │   ├── is-stream@1.1.0 
23:23:33 │ │ │   ├── npm-run-path@1.0.0 
23:23:33 │ │ │   ├── object-assign@4.1.1 
23:23:33 │ │ │   ├── path-key@1.0.0 
23:23:33 │ │ │   └── strip-eof@1.0.0 
23:23:33 │ │ └─┬ widest-line@1.0.0 
23:23:33 │ │   └─┬ string-width@1.0.2 
23:23:33 │ │     ├── code-point-at@1.1.0 
23:23:33 │ │     ├─┬ is-fullwidth-code-point@1.0.0 
23:23:33 │ │     │ └── number-is-nan@1.0.1 
23:23:33 │ │     └─┬ strip-ansi@3.0.1 
23:23:33 │ │       └── ansi-regex@2.1.1 
23:23:33 │ ├─┬ chalk@1.1.3 
23:23:33 │ │ ├── ansi-styles@2.2.1 
23:23:33 │ │ ├── escape-string-regexp@1.0.5 
23:23:33 │ │ ├─┬ has-ansi@2.0.0 
23:23:33 │ │ │ └── ansi-regex@2.1.1 
23:23:33 │ │ ├─┬ strip-ansi@3.0.1 
23:23:33 │ │ │ └── ansi-regex@2.1.1 
23:23:33 │ │ └── supports-color@2.0.0 
23:23:33 │ ├─┬ configstore@3.1.0 
23:23:33 │ │ ├─┬ dot-prop@4.1.1 
23:23:33 │ │ │ └── is-obj@1.0.1 
23:23:33 │ │ ├─┬ make-dir@1.0.0 
23:23:33 │ │ │ └── pify@2.3.0 
23:23:33 │ │ └─┬ unique-string@1.0.0 
23:23:33 │ │   └── crypto-random-string@1.0.0 
23:23:33 │ ├── import-lazy@2.1.0 
23:23:33 │ ├── is-npm@1.0.0 
23:23:33 │ ├─┬ latest-version@3.1.0 
23:23:33 │ │ └─┬ package-json@4.0.1 
23:23:33 │ │   ├─┬ got@6.7.1 
23:23:33 │ │   │ ├─┬ create-error-class@3.0.2 
23:23:33 │ │   │ │ └── capture-stack-trace@1.0.0 
23:23:33 │ │   │ ├── duplexer3@0.1.4 
23:23:33 │ │   │ ├── get-stream@3.0.0 
23:23:33 │ │   │ ├── is-redirect@1.0.0 
23:23:33 │ │   │ ├── is-retry-allowed@1.1.0 
23:23:33 │ │   │ ├── is-stream@1.1.0 
23:23:33 │ │   │ ├── lowercase-keys@1.0.0 
23:23:33 │ │   │ ├── timed-out@4.0.1 
23:23:33 │ │   │ ├── unzip-response@2.0.1 
23:23:33 │ │   │ └─┬ url-parse-lax@1.0.0 
23:23:33 │ │   │   └── prepend-http@1.0.4 
23:23:33 │ │   ├─┬ registry-auth-token@3.3.1 
23:23:33 │ │   │ └─┬ rc@1.2.1 
23:23:33 │ │   │   ├── deep-extend@0.4.2 
23:23:33 │ │   │   ├── minimist@1.2.0 
23:23:33 │ │   │   └── strip-json-comments@2.0.1 
23:23:33 │ │   └─┬ registry-url@3.1.0 
23:23:33 │ │     └─┬ rc@1.2.1 
23:23:33 │ │       ├── deep-extend@0.4.2 
23:23:33 │ │       ├── minimist@1.2.0 
23:23:33 │ │       └── strip-json-comments@2.0.1 
23:23:33 │ ├── semver-diff@2.1.0 
23:23:33 │ └── xdg-basedir@3.0.0 
23:23:33 ├── uuid@3.1.0 
23:23:33 ├─┬ which@1.3.0 
23:23:33 │ └── isexe@2.0.0 
23:23:33 ├─┬ worker-farm@1.5.0 
23:23:33 │ ├─┬ errno@0.1.4 
23:23:33 │ │ └── prr@0.0.0 
23:23:33 │ └── xtend@4.0.1 
23:23:33 └─┬ write-file-atomic@2.3.0 
23:23:33   └── signal-exit@3.0.2 
23:23:33 
23:23:33 [  8%] Building Javascript files
23:23:33 cd /tmp/binarydeb/ros-kinetic-rospilot-1.3.7/share/web_assets && /tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu/node_modules/npm/bin/npm-cli.js install
23:23:34 npm ERR! Buffer.alloc is not a function
23:23:34 
23:23:34 npm ERR! A complete log of this run can be found in:
23:23:34 npm ERR!     /home/buildfarm/.npm/_logs/2017-09-13T06_23_34_494Z-debug.log
23:23:34 make[4]: *** [DIRECTORY] Error 1
23:23:34 CMakeFiles/javascript_assets.dir/build.make:65: recipe for target 'DIRECTORY' failed
23:23:34 make[4]: Leaving directory '/tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu'
23:23:34 CMakeFiles/Makefile2:144: recipe for target 'CMakeFiles/javascript_assets.dir/all' failed
23:23:34 make[3]: Leaving directory '/tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu'
23:23:34 make[3]: *** [CMakeFiles/javascript_assets.dir/all] Error 2
23:23:34 make[2]: *** [all] Error 2
23:23:34 Makefile:130: recipe for target 'all' failed
23:23:34 make[2]: Leaving directory '/tmp/binarydeb/ros-kinetic-rospilot-1.3.7/obj-x86_64-linux-gnu'
23:23:34 dh_auto_build: make -j1 returned exit code 2
23:23:34 	cd /tmp/binarydeb/ros-kinetic-rospilot-1.3.7
23:23:34 debian/rules:36: recipe for target 'override_dh_auto_build' failed
23:23:34 make[1]: Leaving directory '/tmp/binarydeb/ros-kinetic-rospilot-1.3.7'
23:23:34 make[1]: *** [override_dh_auto_build] Error 2
23:23:34 make: *** [build] Error 2
23:23:34 debian/rules:23: recipe for target 'build' failed
23:23:34 dpkg-buildpackage: error: debian/rules build gave error exit status 2
23:23:34 E: Building failed
23:23:34 Traceback (most recent call last):
23:23:34 # END SUBSECTION
```

**Comment by cberner** [comment]
Hmm, thanks for the report! Will take a look

**Comment by cberner** [comment]
Ah, I had a commit to fix this, but hadn't released it yet. I just released `1.3.8` which should fix this
